src/unix/fs.c

```diff
@@ -929,6 +929,11 @@ static ssize_t uv__fs_sendfile(uv_fs_t* req) {
         } else if (r == -1 && errno == ENOTSUP) {
           /* ENOTSUP - it could work on another file system type */
           errno = 0;
+        } else if (r == -1 && errno == EXDEV) {
```

Comment:
I think you can combine this with the previous `else if`:

```
} else if (r == -1 && (errno == ENOTSUP || errno == EXDEV)) {
```

Reply:
@cjihrig I combined the checks. PTAL.